src/com/amazon/corretto/crypto/provider/PublicUtils.java

```diff
@@ -0,0 +1,27 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package com.amazon.corretto.crypto.provider;
```

Comment:
question for reviewers -- should this live in a different package from AmazonCorrettoCryptoProvider?

```
package com.amazon.corretto.crypto.utils;

public class PublicUtils { ... }
```

Reply:
I like this.